include/adm/parse.hpp

```diff
@@ -53,6 +51,19 @@ namespace adm {
       const std::string& filename,
       xml::ParserOptions options = xml::ParserOptions::none);
 
+  /**
+   * @brief Parse an xml document containing an audioFormatExtended
+   * node into an adm::Document, using a SADM FrameHeader to check for consistency.
+   * Primarily intended for parsing sadm frames.
+   *
+   * Convenience wrapper for files using `parseXml(std::istream&)`
```

Comment:
this should reference the sadm equivalent

Reply:
b5e13ce572dee4b2864bbcf4f1d69bcb128e3021